### PR TITLE
Globals and Radio

### PR DESCRIPTION
**Pull request by photon1503**
- Created new class Globals to hold variables which are needed in global context
- Extracted Radio from iRacing
- Tested with newest branch of Dashboard (TIME1edit) on real rig

### CODE REVIEW

**Comment by andreasdahl1987** [comment]
@photon1503 
Are you adding anything to this? Hope to get time to test in the evening :)

**Comment by photon1503** [comment]
@andreasdahl1987  I have some changes on another branch, but I think we should stabilize the project. Christmas is coming, so let's stay calm :-)

**Comment by andreasdahl1987** [comment]
Good idea :) I'll see if I can finish the estimated lap time algorithm, and then add the new TIME1 as release. I'll work on adding BMW LMDh, GR96 and W13 support during christmas!

File: iRacing/iRacing.cs [hunk 1]
```diff
@@ -3834,7 +3788,7 @@ public override void DataUpdate()
 
                 double? behindGap = 0;
                 string behindName = "";
-                behindGlobal = behindName;
+                Globals.behindGlobal = behindName;
```
Comment (andreasdahl1987) [inline iRacing/iRacing.cs:3791]: Should also move aheadGlobal to Globals.cS?

Comment (photon1503) [reply iRacing/iRacing.cs:3791]: I think we should move the variables as we need them during refactoring. No need to move them to Globals in advance

Comment (andreasdahl1987) [reply iRacing/iRacing.cs:3791]: I see aheadGlobal is already moved to Globals.cs. Just that you're still using the "local" aheadGlobal in iRacing.cs.  Line 152 and 3777.

Comment (photon1503) [reply iRacing/iRacing.cs:3791]: oh... thx for the hint. Shall I change it?

Comment (photon1503) [reply iRacing/iRacing.cs:3791]: fixed

Comment (photon1503) [reply iRacing/iRacing.cs:3788]: ... on the wrong branch ... now on correct one

**Pull request merged**